frontend/src/__tests__/ViewRoom/index.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import ViewRoom from '../../components/ViewRooms';
+
+describe('ViewRoom', () => {
+  it('Renders correctly', () => {
+    const wrapper = shallow(<ViewRoom />);
+    expect(wrapper.html()).toMatchSnapshot();
```

Comment:
According to the snapshot, it is an empty render. Would it make sense to test some element rendering, or replace the test with an `.emptyRender()`?

Reply:
Did done didit